extensions/quality/json-schema/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://linz.github.io/STAC/extensions/quality/v1.0.0/schema.json#",
+  "description": "STAC Quality Extension for STAC Collections.",
+  "oneOf": [
+    {
+      "$comment": "This is the schema for LINZ STAC Collections.",
+      "allOf": [
+        {
+          "$ref": "https://schemas.stacspec.org/v1.0.0/collection-spec/json-schema/collection.json#"
+        },
+        {
```

Comment:
This looks like it shouldn't be in here. Isn't it already in the 'linz' extension?

Reply:
Sorry, I think I didn't highlight it properly. I meant this whole bit:

```
       {
          "type": "object",
          "required": ["title"],
          "properties": {
            "title": {
              "type": "string",
              "minLength": 1
            }
          }
        }
```